### DISCUSSION TITLE
Could not find the Attributes tab when clicking on the user

### DISCUSSION THREAD

**Discussion by hunglsxx** [keycloak/keycloak#30586] [Q&A]
I want to configure "Custom Attribute in Keycloak Access Token"
I have followed these steps:

1. Open the Keycloak administration console on http://localhost:8080, click on master in the top left corner, create realm, set realm name to mediamtx, Save.

2. Open the Client scopes page, create client scope, set name to mediamtx, Save.

3. Open the Mappers tab, configure a new Mapper, User Attribute.
    Name: mediamtx_permissions
    User Attribute: mediamtx_permissions
    Token Claim Name: mediamtx_permissions
    Claim JSON Type: JSON
    Multivalued: On
    Save

5. Open the Clients page, create client, set Client ID to mediamtx, Next, Client authentication On, Next, Save.

6. Open the Credentials tab, copy the client secret somewhere.

7. Open the Client scopes tab, add client scope, select mediamtx, Add, Default.

8. Open the Users page, create user, set Username to testuser, open the Credentials tab, set password, pick a password, Save.

9. Open the Attributes tab, add an attribute.

    Key: mediamtx_permissions
    Value: {"action":"publish", "paths": "all"}
    You can add as many attributes with the key mediamtx_permissions as you want, each with a single permission in it.

However, in step 8, I could not find the Attributes tab when clicking on the user.

<img width="1199" alt="Screenshot 2024-06-19 at 22 53 38" src="https://github.com/keycloak/keycloak/assets/50700597/2e53a32a-10cc-49b0-8159-a9e7ca0c8967">

I am using version 25.0.0 on Docker.
I have referred to many guides from various sources, all of which have the Attributes tab, but I do not. Please help!!

**Comment by Xulunix** [comment]
Custom attributes are now visible in the "Details" tab.
The documentation is probably outdated

**Comment by codespearhead** [comment]
Are you trying to set [unmanaged attributes](https://www.keycloak.org/docs/latest/server_admin/#_understanding-managed-and-unmanaged-attributes)?

If so, you have to enable them first as per the documentation:

https://github.com/keycloak/keycloak/blob/4be058a3dd64deebaa5a1397fdf1a8b5f74f99aa/docs/documentation/server_admin/topics/users/user-profile.adoc?plain=1#L91-L92

Duplicate: https://github.com/keycloak/keycloak/discussions/28316
